lib/consensus.js

```diff
@@ -39,6 +40,26 @@ function isValidBlockHeader(newHeader, previousHeaders, network = 'mainnet') {
     && hasGreaterThanMedianTimestamp(newHeader, previousHeaders);
 }
 
+/**
+ * validates an array of tx hashes or objects
+ * against a merkleblock and the local header chain
+ * @param {Object[]|string[]} transactions
+ * @param {Object} merkleBlock - a MerkleBlock object
```

Comment:
Are we getting plain object or MerkleBlock instance? The same about transactions

Reply:
It should be instances. Updated accordingly with latest commit.

Reply:
Then we can specify types MerkleBlock and Transaction instead of Object

Reply:
agree.